modules/tiles/src/tileset/tileset-3d.ts

```diff
@@ -540,7 +579,14 @@ export class Tileset3D {
     // Sort requests by priority before making any requests.
     // This makes it less likely this requests will be cancelled after being issued.
     // requestedTiles.sort((a, b) => a._priority - b._priority);
+
+    // Stop loading tiles if memory was exceeded
+    this._memoryExceeded = false;
+
     for (const tile of this._requestedTiles) {
+      if (this._memoryExceeded) {
+        break;
```

Comment:
This seems subtle... Should we log something? Add a stat indicating that we have rejected a tile?

Reply:
I don't think this is really rejecting the tile, but more like it stops loading downloaded tiles to GPU memory in case it already exceeded the limit. The tile would later be loaded again when there is a chance. The corresponding Cesium part is [here](https://github.com/CesiumGS/cesium/blob/6e8b5d8fba959dd4df92c1aa4c69c6a62900aa0d/packages/engine/Source/Scene/Cesium3DTileset.js#L2610). But I have to say that so far in my tests I wasn't able to reach that `memoryExceeded` condition in loaders.gl. It might be that these get sorted out of the requester before they are reached. We could try to just remove this functionality?